containers/wordpress/wp_base/https/files/ssl-params.conf

```diff
@@ -0,0 +1,12 @@
+ssl_protocols TLSv1.2;
+ssl_prefer_server_ciphers on;
+ssl_dhparam /home/base/certificates/ssl/dhparam.pem;
+ssl_ciphers ECDHE-RSA-AES128-GCM-SHA256;
+ssl_session_timeout  10m;
+ssl_session_cache shared:SSL:10m;
+resolver 8.8.8.8 8.8.4.4 valid=300s;
+resolver_timeout 5s;
+add_header X-Frame-Options DENY;
+add_header X-Content-Type-Options nosniff;
+add_header X-XSS-Protection "1; mode=block";
+
```

Comment:
This whole file is redundant in the latest version of PKB, we should do the same for simplicity.

Reply:
Do you mean it is not required any more?